src/index.jsx

```diff
@@ -61,42 +127,57 @@ const socialLinks = [
 
 
 const App = () => (
-  <IntlProvider locale="en">
-    <SiteFooter
-      siteName="edX"
-      siteLogo={FooterLogo}
-      marketingSiteBaseUrl="https://www.example.com"
-      enterpriseMarketingLink={{
-        url: 'https://business.edx.org',
-        queryParams: { utm_test: 'utm_test_value' },
-      }}
-      supportUrl="https://www.example.com/support"
-      contactUrl="https://www.example.com/contact"
-      openSourceUrl="https://www.example.com/open"
-      termsOfServiceUrl="https://www.example.com/terms-of-service"
-      privacyPolicyUrl="https://www.example.com/privacy-policy"
-      appleAppStoreUrl="https://store.apple.com"
-      googlePlayUrl="https://play.google.com"
-      handleAllTrackEvents={() => {}}
-      socialLinks={socialLinks}
-      supportedLanguages={[
-        {
-          label: 'English',
-          value: 'en',
-        }, {
-          label: 'español',
-          value: 'es-419',
-        },
-      ]}
-      languageForm={{
-        activeLanguage: 'en',
-        screenReaderLabel: 'Choose Language',
-        submitLabel: 'Apply',
-        icon: <FontAwesomeIcon icon={faLanguage} size="2x" className="text-primary" />,
-        onLanguageSelected: () => {},
-      }}
-    />
-  </IntlProvider>
+  <SiteFooter
+    siteLogo={{
+      src: FooterLogo,
+      altText: 'edx Logo',
+      ariaLabel: 'edX Home',
+    }}
+    ariaLabel="Page Footer"
+    marketingSiteBaseUrl="https://www.example.com"
+    appleAppStore={{
+      url: 'https://store.apple.com',
+      altText: 'Download the edX mobile app from the Apple App Store',
+    }}
+    googlePlay={{
+      url: 'https://play.google.com',
+      altText: 'Download the edX mobile app from Google Play',
+    }}
+    handleAllTrackEvents={() => {}}
+    linkSectionOne={{
+      title: 'edX',
+      linkList: edXLinks,
+    }}
+    linkSectionTwo={{
+      title: 'Legal',
+      linkList: legalLinks,
+    }}
+    linkSectionThree={{
+      title: 'Connect',
+      linkList: connectLinks,
+    }}
+    socialLinks={socialLinks}
+    supportedLanguages={[
+      {
+        label: 'English',
+        value: 'en',
+      }, {
+        label: 'español',
+        value: 'es-419',
+      },
+    ]}
+    languageForm={{
+      activeLanguage: 'en',
+      screenReaderLabel: 'Choose Language',
+      submitLabel: 'Apply',
+      icon: <FontAwesomeIcon icon={faLanguage} size="2x" className="text-primary" />,
+      onLanguageSelected: () => {},
+    }}
+    copyright="© 2012–2019 edX Inc."
+    trademark={(
+      <React.Fragment>EdX, Open edX, and MicroMasters are registered trademarks of edX Inc. | 深圳市恒宇博科技有限公司 <a href="http://www.beian.miit.gov.cn">粤ICP备17044299号-2</a></React.Fragment>
```

Comment:
Personal preference but the shorthand for fragments is great if you have seen it before:

```jsx
<>
  <div />
  <div />
</>
```